specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/stable/2021-06-01/postgresql.json

```diff
@@ -1509,6 +1440,146 @@
+          "format": "int32",
+          "description": "Backup retention days for the server."
+        },
+        "geoRedundantBackup": {
+          "type": "string",
+          "description": "A value indicating whether Geo-Redundant backup is enabled on the server.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "GeoRedundantBackupEnum",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "earliestRestoreDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The earliest restore point time (ISO8601 format) for server.",
+          "readOnly": true
+        }
+      },
+      "description": "Backup properties of a server"
+    },
+    "Network": {
+      "properties": {
+        "publicNetworkAccess": {
+          "type": "string",
+          "description": "public network access is enabled or not",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "ServerPublicNetworkAccessState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "delegatedSubnetResourceId": {
+          "type": "string",
+          "description": "delegated subnet arm resource id.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "privateDnsZoneArmResourceId": {
+          "type": "string",
+          "description": "private dns zone arm resource id.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        }
+      },
+      "description": "Network properties of a server"
+    },
+    "HighAvailability": {
+      "properties": {
+        "mode": {
+          "type": "string",
+          "description": "The HA mode for the server.",
+          "enum": [
+            "Disabled",
+            "ZoneRedundant"
+          ],
+          "x-ms-enum": {
+            "name": "HighAvailabilityMode",
+            "modelAsString": true
+          }
+        },
+        "state": {
+          "type": "string",
+          "description": "A state of a HA server that is visible to user.",
+          "enum": [
+            "NotEnabled",
+            "CreatingStandby",
+            "ReplicatingData",
+            "FailingOver",
+            "Healthy",
+            "RemovingStandby"
+          ],
+          "x-ms-enum": {
+            "name": "ServerHAState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "standbyAvailabilityZone": {
+          "type": "string",
+          "description": "availability zone information of the standby.",
+          "readOnly": true
+        }
+      },
+      "description": "High availability properties of a server"
+    },
+    "MaintenanceWindow": {
```

Comment:
Looks lots of definition is disorder in new version(Like MaintenanceWindow). it make review more difficult. can we reorder it?